### PR TITLE
DOCS-6252 create connected app with UI

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Option to create a connected app with UI is not present, so this info was removed in DOCS-6172 and https://github.com/mulesoft/docs-api-community-manager/pull/115 . It will be released Oct 19, so merge this PR as part of that release.

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
cc @jdeoliveira - Today I removed the info on creating a connected app with the UI, and I will add it back at the time of the Oct 19 release.

File: modules/ROOT/pages/configure-acm.adoc [hunk 1]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
 
 === Create a Connected App [[create-a-connected-app]]
 
-To create a connected app with an API call, follow this example using `cURL`.
+You can perform this step either with the user interface or with an API call.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
You can perform this step by using either the user interface or an API call.
```

File: modules/ROOT/pages/configure-acm.adoc [hunk 2]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
...
 
 Your account must have the Org Admin permission.
 
+==== Create a Connected App with the User Interface [[create-a-connected-app-with-the-user-interface]]
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
==== Create a Connected App by Using the User Interface [[create-a-connected-app-with-the-user-interface]]
```

File: modules/ROOT/pages/configure-acm.adoc [hunk 3]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
...
+
+To create a connected app with the user interface, in Anypoint Platform, navigate to *Access Management* > *Applications* > *Create New Application*, and provide this information.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
To create a connected app by using the user interface, in Anypoint Platform, navigate to *Access Management* > *Applications* > *Create New Application*, and provide this information:
```

File: modules/ROOT/pages/configure-acm.adoc [hunk 4]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
...
+
+. Enter the *Application Name*.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
. Enter a value for *Application Name*.
```

File: modules/ROOT/pages/configure-acm.adoc [hunk 5]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
...
++
+Find this URL in *Salesforce Setup* > *Auth. Providers* > *Anypoint* > *Callback URL*.
++
+. Set *User Grant Types*.
++
+Select *Authorization Code*, *Refresh Token*, *Full Access*, and *Background Access*.
++
+. Click *Create Application*.
+
+==== Create a Connected App with an API Call [[create-a-connected-app-with-an-api-call]]
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
==== Create a Connected App by Using an API Call [[create-a-connected-app-with-an-api-call]]
```

File: modules/ROOT/pages/configure-acm.adoc [hunk 6]
```diff
@@ -61,10 +63,29 @@ The external data source specifies how ACM components and data objects interact
...
+
+To create a connected app with an API call, follow this example using `cURL`.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-acm.adoc]:
```suggestion
To create a connected app by using an API call, follow this example using `cURL`operation:
```

Comment (kbostrom) [reply modules/ROOT/pages/configure-acm.adoc]: Followed by steps?

**Review by kbostrom — APPROVED** [review]
Done!

File: modules/ROOT/pages/configure-acm.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/configure-acm.adoc]: Fixed. Thank you!

**Pull request merged**